### PR TITLE
[ONNX] Fix export of constant dim

### PR DESCRIPTION
**Pull request by neginraoof**
Fix export of constant dim

### CODE REVIEW

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -2953,6 +2953,9 @@ def _weight_norm(g, weight_v, weight_g, dim):
 def dim(g, self):
     """Implement the dim functionality available for a pytorch tensor in ONNX"""
     # ONNX does not support dim directly in this opset so we can use 2 ops to get the info
+    sizes = sym_help._get_tensor_sizes(self)
+    if sizes is not None:
+        return g.op("Constant", value_t=torch.tensor(len(sizes)))
```
Comment (shubhambhokare1) [inline torch/onnx/symbolic_opset9.py:2958]: Do we need a test for this?

**Comment by neginraoof** [comment]
Will be handled in: https://github.com/pytorch/pytorch/pull/59122
